doc/tutorials/gapi/face_beautification/face_beautification.markdown

```diff
@@ -0,0 +1,500 @@
+# Implementing a face beautification algorithm on G-API {#tutorial_gapi_face_beautification}
+
+@prev_tutorial{tutorial_gapi_anisotropic_segmentation}
+
+[TOC]
+
+# Introduction {#gapi_fb_intro}
+
+In this tutorial you are going to learn:
+* How to create a custom kernel implementation (using OpenCV backend);
+* How to use Inference-API to infer networks inside a graph;
+* How to process a videostream inside a graph using Streaming-API;
+* The main ideas of the Face Beautification algorithm.
+
+You can find source code in the `modules/gapi/samples/face_beautification.cpp`
+of the OpenCV source code library.
+
+# New G-API features description {#gapi_fb_features}
+
+Before we can discuss an implementation of any algorithm, we should
+sort out new G-API features applied there. Obviously, paragraphs below are not
+detailed documentary articles, they are just brief guides how to use one feature
+or another with examples of usage.
```

Comment:
Please, get rid of repetition here: `use ... with examples of usage`

Reply:
fixed